solidity/test/cross-chain/wormhole/L2BTCRedeemerWormhole.test.ts

```diff
@@ -219,7 +219,7 @@ describe("L2BTCRedeemerWormhole", () => {
           l2BtcRedeemer
             .connect(governance)
             .updateMinimumRedemptionAmount(ethers.constants.Zero)
-        ).to.be.revertedWith("Minimum redemption amount must not be 0")
```

Comment:
Nitpick: Would something like this work?

```
.to.be.revertedWithCustomError(contract, "RedemptionTooLow)
```

Reply:
Addressed in [chore: update redeemer tests reversion handling](https://github.com/threshold-network/tbtc-v2/pull/882/commits/50664eacaeb0eb8c81accf2927834f394401e7d6)